src/test/java/edu/cornell/kfs/concur/eventnotification/rest/jsonObjects/ConcurDTOJsonTest.java

```diff
@@ -0,0 +1,124 @@
+package edu.cornell.kfs.concur.eventnotification.rest.jsonObjects;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.cornell.kfs.concur.rest.jsonObjects.ConcurOauth2TokenResponseDTO;
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.util.CUJsonUtils;
+
+public class ConcurDTOJsonTest {
+    private static final String TOKEN_TYPE = "token_type";
+    private static final String SCOPE = "scope";
+    private static final String REFRESH_TOKEN = "refresh_token";
+    private static final String REFRESH_EXPIRES_IN = "refresh_expires_in";
+    private static final String ID_TOKEN = "id_token";
+    private static final String GEOLOCATION = "geolocation";
+    private static final String EXPIRES_IN = "expires_in";
+    private static final String ACCESS_TOKEN = "access_token";
+    private static final Logger LOG = LogManager.getLogger();
+    public static final String OAUTH2_TOKEN_RESPONSE_FILE_NAME = "src/test/resources/edu/cornell/kfs/concur/rest/jsonObjects/fixture/ConcurOauth2TokenResponse.json";
+    public static final String OAUTH2_TOKEN_RESPONSE_WITH_EXTRA_VALUES_FILE_NAME = "src/test/resources/edu/cornell/kfs/concur/rest/jsonObjects/fixture/ConcurOauth2TokenResponseWithExtraValues.json";
+    public static final String EXPECTED_TOKEN_VALUES_FILE_NAME = "src/test/resources/edu/cornell/kfs/concur/rest/jsonObjects/fixture/jsonValues.txt";
+    
+    private ObjectMapper objectMapper;
+    private Map<String, Object> exptectedJsonValues;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        objectMapper = CUJsonUtils.buildObjectMapperUsingDefaultTimeZone();
+        buildExptectedJsonValues();
+    }
+    
+    private void buildExptectedJsonValues() {
+        exptectedJsonValues = new HashMap<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(EXPECTED_TOKEN_VALUES_FILE_NAME));) {
```

Comment:
Should the FileReader be created in a separate statement within the same try-with-resources entry block, so that the it can be explicitly auto-closed afterwards?